source/index.html

```diff
@@ -327,16 +322,16 @@ <h3>Sleeping Options:</h3>
             <h3>Cat Boarding:</h3>
             <ul>
               <li>
-                5x12 luxury suite with perches, fountain and more $19 One cat
-                $25 for 2
+                5x12 luxury suite with perches, fountain and more. $24 One cat
+                $10 per each additional cat.
```

Comment:
```suggestion
                5x12 luxury suite with perches, fountain and more. One cat - $24, additional cats - $10 per cat.
```

Just to sort of make it match the formatting above?